### PR TITLE
First Feature Model Visualization Prototype

### LINKED ISSUE #69: Implement Transpiler in proper language (G6)

### PR DESCRIPTION
**Pull request by wurstbroteater**
Current state for this feature is "not good for production but enough for first prototype". 
There is a list of feature requests and known bugs in issue #79. If you find more, feel free to add them!

Transpiler and feature model visualization are currently independent of backend and frontend. 
To connect backend and transpiler (see issue #80) we probably need a more advanced version of file upload. This needs to be discussed with the frontend team

## Start
- Navigate to `backend/transpiler/` and in this folder:
- Run `python tests.py`. This will create 4 files starting with _g6_model_  and 1 file named _model_npc.json_ in `backend/transpiler/xmlExamples`
- in `backend/transpiler/index.html`  in Line 16 modify the model you want to display 
-  Run `npx serve`

### CODE REVIEW

File: backend/requirements.txt [hunk 1]
```diff
@@ -13,3 +13,4 @@ PyJWT==2.3.0
 pytz==2021.3
 six==1.16.0
 sqlparse==0.4.2
+texttable==1.6.4
```
Comment (Eixix) [inline backend/requirements.txt]: Used for?

File: backend/.dockerignore [hunk 2]
```diff
@@ -1,4 +0,0 @@
-Dockerfile
```
Comment (Eixix) [inline backend/.dockerignore:1]: Do not delete

**Review by Eixix — CHANGES_REQUESTED** [review]
Dont delete .dockerignore

**Review by Bishares — DISMISSED** [review]
bassd

**Review by Eixix — APPROVED** [review]
Subber

**Pull request merged**